lib/rzo/app/templates/Vagrantfile.erb

```diff
@@ -16,6 +16,9 @@ Vagrant.configure(2) do |config|
     cfg.vm.box_url = <%= nc['box_url'].inspect %>
     cfg.vm.box_download_checksum = <%= nc['box_download_checksum'].inspect %>
     cfg.vm.box_download_checksum_type = <%= nc['box_download_checksum_type'].inspect %>
+    <%- if nc['boot_timeout'] -%>
```

Comment:
Awesome! Thank you!

Could you please add documentation under https://github.com/ghoneycutt/rizzo#nodes and mention that Vagrant's default is 300 seconds and that the value should be denoted in seconds.